build/Tools/xsd2code/Xsd2Code.exe.config

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0"?>
+<configuration>
+<startup><supportedRuntime version="v2.0.50727"/></startup></configuration>
```

Comment:
@jaffinito I know this PR has merged already, though I'm fairly confident that Xsd2Code will still run fine if we change this to `v4.0.30319`. That way people don't need to install .NET 3.5 to run it.